script.js

```diff
@@ -0,0 +1,36 @@
+name  = prompt("Hey, what's your name?")
+age = prompt("How old are you?")
+
+
+
+for(var i = age; i <= 17; i++) {
+name = "Young person (you have played the lottery illegally)";
+}
+
+
+
+if (name !== null) {
+  var entryPath = prompt(name + ", You have just won 2 billion dollars from playing the lottery. Now that you are filthy rich, what will you do first? Here are your options: 1. Pay off your debt, the debt of your immediate family, and set up some savings accounts 2. Go to Vegas and make it rain for a few days while staying in a penthouse suite. Please enter '1' or '2'")
+
+  if (entryPath == "1") {
+    var paidOffDebts = prompt("You and your immediate family members are officially debt free. Other random far-away family members (cousins of cousins twice removed) are now all asking for a handout. Not really a handout but pretty much demanding money or else they will hire a contract killer to harm you or put your personal info on the internet (racy old facebook pictures and credit card information) Will you 1. Decide to pay off these mean semi-family members? or 2. Say F*ck them and keep it moving- buy yourself a nice house, cool cars, and hire a great security team? Please enter '1' or '2'")
+        if (paidOffDebts == '1') {
+          console.log ("This cycle never ended and people kept asking you for money. You eventually gave away all of your 2 billion dollar fortune. Unfortunately, you are now struggling to survive.")
+        }
+        else if (paidOffDebts == '2') {
+          console.log("The security team you hired paid off and no one has threatened you or tried to harm you since. You also hired a great lawyer who has your back no matter what. You are still very rich and occassionally go on a shopping spree or two but you have kept a majority of your wealth. You are happy!")
+        }
+    }
+
+
+else if (entryPath == "2") {
```

Comment:
This block should be indented to be at the same level as the other 'entryPath' condition on line 15